test/prettier-setup.test.ts

```diff
@@ -0,0 +1,101 @@
+import { describe, it, expect } from "vitest";
+import fs from "fs";
+
+describe("Prettier config static test", () => {
```

Comment:
These kind of configurations should also be tested in a E2E or integration test on the "use" of the template but ensuring that its expected output for now is ok for now.